code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -163,12 +223,202 @@
 	var/mob/living/carbon/human/C = owner
 	var/mob/camera/aiEye/remote/xenobio/remote_eye = C.remote_control
 	var/obj/machinery/computer/camera_advanced/xenobio/X = target
-
+	var/obj/machinery/monkey_recycler/recycler = X.connected_recycler
+	
+	if(!recycler)
+		to_chat(owner, "<span class='notice'>There is no connected monkey recycler.  Use a multitool to link one.</span>")
+		return
 	if(cameranet.checkTurfVis(remote_eye.loc))
 		for(var/mob/living/carbon/human/M in remote_eye.loc)
 			if(issmall(M) && M.stat)
 				M.visible_message("[M] vanishes as [M.p_theyre()] reclaimed for recycling!")
-				X.monkeys = round(X.monkeys + 0.2,0.1)
+				recycler.use_power(500)
+				X.monkeys = round(X.monkeys + recycler.cube_production/recycler.required_grind, 0.1)
```

Comment:
TG's math on this is a lot better.

Reply:
Except their cube production system is different then ours, rather than modifying the number of monkeys required per cube, they have each "cube" only produce 0.25 cubes per monkey. Basically, when you throw a monkey into a tg recyccler, it generates 0.25 cubes. When you put a monkey in a paradise recycler, it just simply doesn't do anything and waits for 3 more monkeys. I did try it with the TG math, only resulting in infinitely reusable monkeys since the cube production on paradise is 1 whereas with TG its 0.25. Obviously the simple solution to this is to simulate the 0.25 monkey cube production of TG by diving the cube production by the # of monkeys it takes to create one cube.